src/autopas/containers/linkedCells/traversals/LCC01Traversal3B.h

```diff
@@ -41,29 +38,29 @@ class LCC01Traversal3B
    * @param functor The functor that defines the interaction of three particles.
    * @param interactionLength Interaction length (cutoff + skin).
    * @param cellLength cell length in CellBlock3D
+   * @param dataLayout The data layout with which this traversal should be initialised.
+   * @param useNewton3 Parameter to specify whether the traversal makes use of newton3 or not.
    * @todo Pass cutoff to _cellFunctor instead of interactionLength, unless this functor is used to build verlet-lists,
    * in that case the interactionLength is needed!
    */
   explicit LCC01Traversal3B(const std::array<unsigned long, 3> &dims, Functor *functor, const double interactionLength,
-                            const std::array<double, 3> &cellLength)
-      : C01BasedTraversal<ParticleCell, Functor, InteractionTypeOption::threeBody, dataLayout, useNewton3, 3>(
-            dims, functor, interactionLength, cellLength),
-        _cellFunctor(functor, interactionLength /*should use cutoff here, if not used to build verlet-lists*/),
+                            const std::array<double, 3> &cellLength, DataLayoutOption dataLayout, bool useNewton3)
+      : TraversalInterface(dataLayout, useNewton3),
+        C01BasedTraversal<ParticleCell, Functor, 3>(dims, functor, interactionLength, cellLength, dataLayout,
+                                                    useNewton3),
+        _cellFunctor(functor, interactionLength /*should use cutoff here, if not used to build verlet-lists*/,
```

Comment:
I _don't think_ the comment is correct and it should always be `interactionLength`

Reply:
Okay, I removed it and also the todo above it, since I don't think this is desirable.